### PR TITLE
Update NEWS for 1.9.1

### PR DESCRIPTION
**Pull request by alexlarsson**
Initial update of the NEWS file for 1.9.1 (although still waiting on an ostree release before it can go out).

### CODE REVIEW

**Comment by alexlarsson** [comment]
I'll do a 1.8.3 release first and rebase the news update on that.

**Comment by alexlarsson** [comment]
Also backported some test changes i did for the 1.8.3 release

File: NEWS [hunk 1]
```diff
@@ -1,3 +1,120 @@
...
+Released: 2020-11-xx
+
+This is the first unstable release in the series that will lead to
+1.10. The main change in this version is a new format for the summary
+file used when accessing an OSTree repository on the network. For this
+reason we now require OSTree version 2020.8.
+
+The new format should make getting the initial metadata required for
+most flatpak operations much faster, and use less network
+bandwitdh. This will allow repostories to scale to more apps and more
```
Comment (AsciiWolf) [inline NEWS:12]: Small typos: "bandwitdh", "repostories", "compatibilty"

Comment (alexlarsson) [reply NEWS:12]: thanks

**Pull request merged**